common/services/conversion/track.ts

```diff
@@ -126,24 +126,27 @@ function trackSegmentEvent({
 
 function track(conversion: Conversion) {
   const debug = getCookie(cookies.analyticsDebug) === true;
+  const toggles = getActiveToggles(getCookies());
```

Comment:
Maybe worth a comment here about why we can't use `useToggles()`? I think it's because we're not in a React hook?